superset/connectors/base/models.py

```diff
@@ -533,6 +533,7 @@ class BaseColumn(AuditMixinNullable, ImportExportMixin):
     verbose_name = Column(String(1024))
     is_active = Column(Boolean, default=True)
     type = Column(String(32))
+    business_type = Column(String(255))
```

Comment:
Hi, @cccs-RyanS... do you want to give more context about this PR before putting work into it? I'm worried that this will need a DB migration, and this columns seems very specific.

Reply:
Hey so my org had discussed this with Ville Brofeldt and I was waiting on some feedback. The idea was to have business types on top of the db types to have added flexibility. Let me speak to my coworker and I can get you a bunch of extra context.